docs/content/reference/command-line-interface/_index.md

```diff
@@ -564,12 +566,35 @@ The following command will also wait on **all** the resources defined in the `dr
 ```
 drasi wait -f drasi-resources.yaml -t 20
 ```
+### drasi watch
+**Purpose**: The `watch` command allows users to continuously monitor the result set of a specified query in real-time.
+
+**Arguments and Flags**:
+- `query name` (required): This argument specifies the name of the query to watch.
```

Comment:
If you are going to include "required" and "optional" annotations, which are an improvement, need to propagate the same to all the other commands. Need consistency throughout docs.